shared/js/project.js

```diff
@@ -234,11 +251,17 @@ var projects = module.exports = {
     current.level = this.appToProjectUrl();
 
     if (channelId && current.isOwner) {
+      current.migratedToS3 = true;
```

Comment:
I would rather see this depend on appOptions.project.migrateToS3, and set that in the appOptions, and then enable that only for Big Game (or possibly template levels), as a means for controlling which levels get migrated. Then below, make the code `if (data.migratedToS3)` not be nested inside code which is specific to embedded levels.